src/web/modules/transactions/fix_async_failed_stripe_refund.ts

```diff
@@ -22,15 +24,30 @@ export async function fixAsyncFailedStripeRefund(req: Request, res: Response, ne
         const chargeId = transaction.parent_transaction_id
         const zendeskTicketId = req.body.zendesk_ticket_id
         const githubUserId = req.user && req.user.username
+        const parentTransaction = await Ledger.transactions.retrieve(transaction.parent_transaction_id)
 
         await Connector.fixAsyncFailedStripeRefund.fixStripeRefund(gatewayAccountId, refundId, chargeId, {
             zendesk_ticket_id: zendeskTicketId,
             github_user_id: githubUserId
         })
-        res.render(`transactions/confirmFixAsyncFailedStripeRefund`, {
-            transaction
+
+        res.render(`transactions/fixAsyncFailedStripeRefundSuccess`, {
+            transaction,
+            parentTransaction,
+            csrf: req.csrfToken()
```

Comment:
Not 100% sure this is required since this page has no forms on it.

Reply:
Good spot will remove it in the next PR